src/main/frontend/commands.cljs

```diff
@@ -278,7 +278,7 @@
     [["Query" [[:editor/input "{{query }}" {:backward-pos 2}]
                [:editor/exit]] query-doc]
      ["Zotero" (zotero-steps) "Import Zotero journal article"]
-     ["Query table function" [[:editor/input "{{function }}" {:backward-pos 2}]] "Create a query table function"]
+     ["Query function" [[:editor/input "{{function }}" {:backward-pos 2}]] "Create a query function"]
```

Comment:
Since we're fixing a bug from #9563, I tweaked the name here since that PR also enhanced the command so that it can be used outside of query tables